loader/loader.go

```diff
@@ -529,7 +529,44 @@ func (l *Loader) Init(ctx context.Context) (err error) {
 		}
 	}
 	if !hasSQLMode {
-		lcfg.To.Session["sql_mode"] = l.cfg.LoaderConfig.SQLMode
+		// When upstream's datatime is 2020-00-00, 2020-00-01, 2020-06-00
```

Comment:
Do we still need to support manually set sql_mode here?

Reply:
What do you mean? I do not understand.